deps/build_OpenBLAS32.v0.3.9.jl

```diff
@@ -0,0 +1,94 @@
+# Generated by running
+# julia --color=yes generate_buildjl.jl O/OpenBLAS/OpenBLAS32@0.3.9/build_tarballs.jl JuliaBinaryWrappers/OpenBLAS32_jll.jl OpenBLAS32-v0.3.9+2
+# in the root the the source tree of https://github.com/JuliaPackaging/Yggdrasil/
+# by first replacing the `include` by its content, see https://github.com/JuliaPackaging/Yggdrasil/issues/858
+# We also added `prefix, ` after `LibraryProduct(`.
+using BinaryProvider # requires BinaryProvider 0.3.0 or later
+# Needed to add this as this method was not defined.
+function BinaryProvider.CompilerABI(; libgfortran_version=nothing)
```

Comment:
@staticfloat Any idea why the generated build.jl used this method ? I don't see it defined in any version of BinaryProvider.

Reply:
Probably because that's the signature of `Pkg.BinaryPlatforms.CompilerABI`